src/node_crypto.cc

```diff
@@ -2542,9 +2542,12 @@ int VerifyCallback(int preverify_ok, X509_STORE_CTX* ctx) {
 }
 
 static bool IsSupportedAuthenticatedMode(int mode) {
+#ifndef OPENSSL_NO_OCB
+  if (mode == EVP_CIPH_OCB_MODE)
+    return true;
```

Comment:
This doesn't seem right. If there is no OCB support, we should not be returning `true` for OCB in a function that determines _supported_ authenticated modes.

Also, I think we should use the same ifdef solution used in the other changes below for consistency.

Reply:
`#ifndef OPENSSL_NO_OCB` implies that there is support for OCB, so this should be correct.